### PR TITLE
Set up structure

### PR DESCRIPTION
**Pull request by somexpert**
To-Do:
- [x] Add changelog
- [x] Add requirements files
- [x] Set up tests project
- [x] Set up docker dev environment
- [x] Add manifest
- [x] Add setup.py
- [x] Move site_views into app
- [x] Fill in description, install_requires, and kewords in setup.py

### CODE REVIEW

**Comment by somexpert** [comment]
@ldko, @damonkelley, this is ready for review.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from setuptools import setup, find_packages
```
Comment (damonkelley) [inline setup.py:2]: We will need a shebang.

**Comment by ldko** [comment]
:+1:

**Comment by damonkelley** [comment]
Can we also make `setup.py` an executable?

File: accession/urls.py [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 from accession.admin_views import duplicates, print_view
-from djangosite.site_views import accession_admin
+from accession.site_views import accession_admin
 
 urlpatterns = patterns('',
     url(r'^databrowse/(.*)', django_databrowse.site.root),
     url(r'^admin/accession/(?P<model_selected>.*)/duplicates/$', duplicates),
     url(r'^admin/([^/]+)/([^/]+)/([0-9]+)/print/$', print_view),
-    url(r'^admin/([^/]+)/([^/]+)/export_csv/$', 'csv_export.views.csv_view', name="tfc_csv_export"),
+   #url(r'^admin/([^/]+)/([^/]+)/export_csv/$', 'csv_export.views.csv_view', name="tfc_csv_export"),
```
Comment (damonkelley) [inline accession/urls.py:10]: Do we _need_ to keep this around?

Comment (somexpert) [inline accession/urls.py:10]: Yes, because that url will return once I replace `csv_export` with `django-queryset-csv`

**Comment by damonkelley** [comment]
:+1:

**Pull request merged**